### ISSUE TITLE
Training Images

### ISSUE THREAD

**Issue by Biriek** [chulminy/Vision_Based_Automated_Crack#1]
Dear Dr. Chul,

I have been trying to learn the codes and understand the outputs. I am now clear with your input and output data. Accordingly, I want to try it with my own images of concrete beam. But, the images have a size of 5500x3500mm and taken with high quality camera. The size and intensity of the image is influencing and terminating the simulation in MATLAB. Would you help me on the procedures to solve the challnges and simplify the simulation. Thanks.

**Comment by chulminy** [comment]
With this information, I can't figure out why your code stopped. Could you show me the pictures or your code?

**Comment by Biriek** [comment]
Here are some of the pictures.

![8_kamera3 21 _dxo](https://user-images.githubusercontent.com/46388016/50766202-a4192e80-1278-11e9-87cd-9b49c114c349.jpg)
![8_kamera3 22 _dxo](https://user-images.githubusercontent.com/46388016/50766203-a4192e80-1278-11e9-8681-4bbb4d627c9e.jpg)
![8_kamera3 23 _dxo](https://user-images.githubusercontent.com/46388016/50766204-a4192e80-1278-11e9-9af1-bafb94bebfea.jpg)
![8_kamera3 24 _dxo](https://user-images.githubusercontent.com/46388016/50766205-a4192e80-1278-11e9-8ac7-b52a608dc2e1.jpg)
![8_kamera3 25 _dxo](https://user-images.githubusercontent.com/46388016/50766206-a4192e80-1278-11e9-8679-ac40e615fee6.jpg)
![8_kamera3 26 _dxo](https://user-images.githubusercontent.com/46388016/50766207-a4b1c500-1278-11e9-93f2-7d15e357bc7a.jpg)
![8_kamera3 27 _dxo](https://user-images.githubusercontent.com/46388016/50766208-a4b1c500-1278-11e9-8936-59ca97c5fddb.jpg)
![8_kamera3 28 _dxo](https://user-images.githubusercontent.com/46388016/50766209-a4b1c500-1278-11e9-8baf-2bd8fbb7d6f4.jpg)
![8_kamera3 29 _dxo](https://user-images.githubusercontent.com/46388016/50766210-a4b1c500-1278-11e9-9da6-cb0f12ece24d.jpg)
![8_kamera3 30 _dxo](https://user-images.githubusercontent.com/46388016/50766211-a4b1c500-1278-11e9-9ccc-445d2deab735.jpg)
![8_kamera3 31 _dxo](https://user-images.githubusercontent.com/46388016/50766212-a54a5b80-1278-11e9-8312-90c871bafc15.jpg)
![8_kamera3 32 _dxo](https://user-images.githubusercontent.com/46388016/50766213-a54a5b80-1278-11e9-8ae4-d74776f8dcd2.jpg)
![8_kamera3 15 _dxo](https://user-images.githubusercontent.com/46388016/50766215-a54a5b80-1278-11e9-9bd5-848cb7695476.jpg)
![8_kamera3 16 _dxo](https://user-images.githubusercontent.com/46388016/50766216-a54a5b80-1278-11e9-8e06-9ce2125abdad.jpg)
![8_kamera3 17 _dxo](https://user-images.githubusercontent.com/46388016/50766217-a54a5b80-1278-11e9-99cd-8c3a75412454.jpg)
![8_kamera3 18 _dxo](https://user-images.githubusercontent.com/46388016/50766218-a5e2f200-1278-11e9-9ec8-ceacb5200f80.jpg)
![8_kamera3 19 _dxo](https://user-images.githubusercontent.com/46388016/50766219-a5e2f200-1278-11e9-9fa9-5983d466459d.jpg)
![8_kamera3 20 _dxo](https://user-images.githubusercontent.com/46388016/50766220-a5e2f200-1278-11e9-88d0-78d68b4758c3.jpg)

**Comment by chulminy** [comment]
One or two images are fine. Please delete the other because it takes a long time of loading.  Do you want to detect crack damage only? If yes, you can only apply the Frangi filter part. You don't need to have object detection and matching process.

**Comment by Biriek** [comment]
Yes, I only want to detect cracks. However, I am very new to MATLAB and have problems with the procedures I need to follow.  I am trying to follow this procedure you mentioned in the repository;
    1. Clone (or download) the source code in github into a directory of your choice.
    2. Manually annotate your object-of-interest using RunLabeling.m.
    3. Run RunTrainClassifier.m and RunTest.m, consecutively. However, you have to modify and update parameters in Parameters.m, applicable to your own application.

The connection between the codes of the individual functions and their sequence to run is confusing me. Would you guide me on the procedures. Thank you very much.

![8_kamera3 21 _dxo](https://user-images.githubusercontent.com/46388016/50768665-7e445780-1281-11e9-9309-3ee6ea822181.jpg)

**Comment by chulminy** [comment]
I'm still confused about what guidance you are asking me. If you want to see the crack detection process, you can put a breakpoint on the corresponding line (i.e. line 278 in RunTest.m) and learn how to detect the crack. The code is to detect crack from each patch. You can replace the patch with your image. However, I'm not sure the code can differentiate between the crack and scratch (watermark) on the specimen in your image. It seems that you are looking for a crack propagated from the bottom of the specimen. You can use that feature for filtering out false-positive detection.

**Comment by KNSd2** [comment]
Dear Dr Chul,

Are the original images and results still available? The link appears to be broken.
kind regards,

**Comment by chulminy** [comment]
Here is the link: 

https://drive.google.com/open?id=1CxjPsvrnSUOnMpZxwdtiH3HTCBwfZoM_

I will update all broken links soon.

**Comment by KNSd2** [comment]
Thanks,

but I still get a 404 error on the google drive link.

**Comment by chulminy** [comment]
It looks like a problem on the issue board. You need to include the '_' at the end. Simply, just copy and paste the link.

**Comment by KNSd2** [comment]
Thanks, that works.
